europython-2020/videos/aaron-ma-machine-learning-for-everyone.json

```diff
@@ -0,0 +1,53 @@
+{
+    "copyright_text": "This video is licensed under the CC BY-NC-SA 3.0 license: https://creativecommons.org/licenses/by-nc-sa/3.0/\nPlease see our speaker release agreement for details: https://ep2020.europython.eu/events/speaker-release-agreement/\n",
+    "description": "Machine learning (ML) is becoming an essential technology for our day to day life. Stop taking ML as a threat and learn it today as not learning it is a HUGE LOSS! Get started today with ML in Aaron's remarkable 45-mins talk. We will begin by talking about the paradigm of ML, then taking a deep dive into Neural Networks and building a Neural Network from scratch with Keras and TensorFlow (the hottest machine learning framework). You'll master the magic of neural networks that are powering incredible advances both in AI, self-driving cars, and much more! Finally, we will finish off by talking about Reinforcement learning and how it is empowering YouTube suggestions along with tips-and-tricks from a specialist plus a grand finale mind-blowing demo. Ready to master the paradigm of ML? Let's get started.",
+    "duration": 2223.0,
+    "language": "eng",
+    "recorded": "2020-07-23",
+    "related_urls": [
+        {
+            "label": "Conference schedule",
+            "url": "https://ep2020.europython.eu/schedule/"
+        },
+        {
+            "label": "Conference Website",
+            "url": "https://ep2020.europython.eu/"
+        },
+        {
+            "label": "https://creativecommons.org/licenses/by-nc-sa/3.0/",
+            "url": "https://creativecommons.org/licenses/by-nc-sa/3.0/"
+        },
+        {
+            "label": "https://ep2020.europython.eu/events/speaker-release-agreement/",
+            "url": "https://ep2020.europython.eu/events/speaker-release-agreement/"
+        },
+        {
+            "label": "Talk URL",
+            "url": "/schedule/24-july?selected=8Tcrj4M-machine-learning-for-everyone"
```

Comment:
Can we change these URLs to have a domain so that they are valid once published to PyVideo?